opwen_email_client/webapp/forms/register.py

```diff
@@ -0,0 +1,87 @@
+from os import getuid
+from pathlib import Path
+from pwd import getpwuid
+from shutil import chown
+
+from flask_wtf import FlaskForm
+from opwen_email_client.webapp.actions import ClientRegister
+from opwen_email_client.webapp.actions import RestartApp
+from opwen_email_client.webapp.config import AppConfig
+from opwen_email_client.webapp.config import root_domain
+from wtforms import StringField
+from wtforms import SubmitField
```

Comment:
Linter was being cranky with this. iSort expects the following:
```suggestion
from flask_wtf import FlaskForm
from wtforms import StringField
from wtforms import SubmitField

from opwen_email_client.webapp.actions import ClientRegister
from opwen_email_client.webapp.actions import RestartApp
from opwen_email_client.webapp.config import AppConfig
from opwen_email_client.webapp.config import root_domain
```

Reply:
I see a lot has been done. What is next from here?

Reply:
@nzola Once Adam implements the suggestions from the code review, we'll be able to merge this and then it will be possible to configure Lokole via a UI as opposed to doing everything on the command line which will enable registering clients on the IIAB deployments.

The next steps would be:

1. Install a LocalOnly Lokole client on RaspberryPi and use this new configuration screen to configure it. Confirm that it sends/receives emails.
2. Test the end-to-end system on IIAB and confirm that we can send/receive emails.

On that note, @adamsclafani are we currently handling the case where someone actually wants to configure a LocalOnly client? I wonder if the form fields in the registration should all be optional and if they're not filled in we'll continue with a LocalOnly setup as opposed to trying to register the client.

Reply:
@c-w @nzola Right now that case is not being handled. It shouldn't be too difficult to add in. Adding a new LocalOnly client registration button might work too